### DISCUSSION TITLE
GM View

### DISCUSSION THREAD

**Discussion by pooya-raz** [fariapp/fari-app#213] [Ideas]
First of all thank you @RPDeshaies and all contributors for your work. We have been having a lot of fun using Fari for our sessions during this pandemic. The clean UI has helped on board new players. 

One thing I wish that Fari had, however, is a GM view. Right now my workflow is that I save all relevant assets in my wiki then cut and paste them to Fari.  Sometimes an NPC enters the scene in the middle, or I use the same NPC across multiple scenes. Since my players (and myself) are new, I reveal nearly all elements of the NPC (such as aspects, skills and stunts) as I feel it empowers them to decided how to best approach the situation. Currently I cut and paste from my wiki to Fari, and it feels a bit clunky. It would be great if we had a GM view where we could prepare and save different assets (such as aspects and NPCs) then add them to scenes when necessary. Also, it would be great if we could create NPCs the same way we create PCs and have clickable skills. This would help with the flow of the game. Thanks!

**Comment by RPDeshaies** [comment]
I think this is a very nice suggestion and would indeed help with GM prep and slowly revealing things.

I will link it to #143 as I feel it's closely related!

> First of all thank you @RPDeshaies and all contributors for your work. We have been having a lot of fun using Fari for our sessions during this pandemic

So pleased you like the app ❤️

**Comment by RPDeshaies** [comment]
Found a way to include that without too much work...!

I'll see if I can include it in the next release

**Reply by pooya-raz** [reply]
That's great! Thank you!
